.github/workflows/validate-upstream.yml

```diff
@@ -31,7 +31,7 @@ jobs:
           repository: docker/docs
       -
         name: Download data files
-        uses: actions/download-artifact@v3
+        uses: actions/download-artifact@v4
```

Comment:
Need to be careful when updating this as v4 of download-artifact is only compatible with its upload-artifact v4 counterpart.

Reply:
So we need to bump upload-artifact in the upstream repos before this one? Is upload-artifact compatible with download v3?

Reply:
Nop must be v4 for both :(

Reply:
That's messed up